spec/salestation/web/ip_address_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Salestation::Web::IPAddress do
+  let(:ip_address) do
+    described_class
+  end
+
+  describe '#extract' do
```

Comment:
This is a class method, so it should be `describe '.extract' do`. `#` is for instance methods, `.` is for class methods

Reply:
fixup: https://github.com/salemove/salestation/pull/67/commits/b5dbdd053b7827fe75ff870f6fd9256f97899792